### ISSUE TITLE
Linux-VirtualBox: error: UUID {} of the medium '.minikube/machines/minikube/disk.vmdk' does not match the value {} stored in the media registry

### ISSUE THREAD

**Issue by tstromberg** [kubernetes/minikube#3198]
https://storage.googleapis.com/minikube-builds/logs/3194/Linux-VirtualBox.txt

Failed at `TestFunctional` and `TestPersistence`:

```shell
Waiting for VM "minikube" to power on...
		}
		STDERR:
		{
		VBoxManage: error: UUID {d85060ec-a1e3-4f74-8fbb-af6e5eb7f25f} of the medium '/home/minikubeci/.minikube/machines/minikube/disk.vmdk' does not match the value {1f88b07b-d3d5-4eb5-b591-9cd530f73e93} stored in the media registry ('/home/minikubeci/.config/VirtualBox/VirtualBox.xml')
		VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component MediumWrap, interface IMedium
		}
		Checking vm logs: /home/minikubeci/.minikube/machines/minikube/minikube/Logs/VBox.log
		E1001 19:13:26.883091    2826 start.go:184] Error starting host: Error starting stopped host: Unable to start the VM: /usr/bin/VBoxManage startvm minikube --type headless failed:
		VBoxManage: error: UUID {d85060ec-a1e3-4f74-8fbb-af6e5eb7f25f} of the medium '/home/minikubeci/.minikube/machines/minikube/disk.vmdk' does not match the value {1f88b07b-d3d5-4eb5-b591-9cd530f73e93} stored in the media registry ('/home/minikubeci/.config/VirtualBox/VirtualBox.xml')
		VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component MediumWrap, interface IMedium
		
		Details: 00:00:00.038076 Power up failed (vrc=VINF_SUCCESS, rc=NS_ERROR_FAILURE (0X80004005)).
		
		 Retrying.
		E1001 19:13:26.883117    2826 start.go:190] Error starting host:  Error starting stopped host: Unable to start the VM: /usr/bin/VBoxManage startvm minikube --type headless failed:
		VBoxManage: error: UUID {d85060ec-a1e3-4f74-8fbb-af6e5eb7f25f} of the medium '/home/minikubeci/.minikube/machines/minikube/disk.vmdk' does not match the value {1f88b07b-d3d5-4eb5-b591-9cd530f73e93} stored in the media registry ('/home/minikubeci/.config/VirtualBox/VirtualBox.xml')
		VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component MediumWrap, interface IMedium
		
		Details: 00:00:00.038076 Power up failed (vrc=VINF_SUCCESS, rc=NS_ERROR_FAILURE (0X80004005))
```

Sounds like there is possibly a conflict with another running process.

**Comment by tstromberg** [comment]
Also:

https://storage.googleapis.com/minikube-builds/logs/3195/Linux-VirtualBox.txt

**Comment by neweyes** [comment]
E1121 15:28:38.864482   85660 start.go:180] Error starting host:  Error starting stopped host: Unable to start the VM: /usr/local/bin/VBoxManage startvm minikube --type headless failed:
VBoxManage: error: UUID {f905783c-fe6b-4910-b310-7e011c548357} of the medium '/Users/inke/.minikube/machines/minikube/disk.vmdk' does not match the value {3afea10b-416d-4385-b8be-47e36d7544ed} stored in the media registry ('/Users/inke/Library/VirtualBox/VirtualBox.xml')
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component MediumWrap, interface IMedium

Details: 00:00:00.043297 Power up failed (vrc=VINF_SUCCESS, rc=NS_ERROR_FAILURE (0X80004005))
[inke@localhost:istio$] minikube start
There is a newer version of minikube available (v0.30.0).  Download it here:
https://github.com/kubernetes/minikube/releases/tag/v0.30.0

To disable this notification, run the following:
minikube config set WantUpdateNotification false
Starting local Kubernetes v1.10.0 cluster...
Starting VM...
Downloading Minikube ISO
 160.27 MB / 160.27 MB [============================================] 100.00% 0s^@
^@^@Getting VM IP address...
Moving files into cluster...
Downloading kubelet v1.10.0

**Comment by neweyes** [comment]
everything be ok after use "rm -rf ~/.minikube"
